### PR TITLE
Fixed GeoJsonExporter and added GeoJSONExportTest

### PR DESCRIPTION
**Pull request by rsupnekar**
Fixes #

Adds a GeoJSON unit test.  Running the test found several issues with GeoJSON export which were also fixed with this issue.

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: sdk/sdk-api/src/main/java/mil/emp3/json/geoJson/GeoJsonExporter.java [hunk 1]
```diff
@@ -408,7 +410,7 @@ private void appendGeoJSONPoint(IFeature feature, StringBuffer buffer) {
 
     public void appendFeature(IFeature feature, StringBuffer buffer) throws IOException {
 
-        buffer.append("{\"type\":  \"Feature\",\n");
+//        buffer.append("{\"type\":  \"Feature\",\n");
```
Comment (jrummel) [inline sdk/sdk-api/src/main/java/mil/emp3/json/geoJson/GeoJsonExporter.java]: Remove comment if it serves no purpose.

File: sdk/sdk-api/src/main/java/mil/emp3/json/geoJson/GeoJsonExporter.java [hunk 2]
```diff
@@ -466,9 +471,14 @@ private void appendFeatureList(List<IFeature> featureList, StringBuffer buffer)
...
-            getAllFeatures(ovl, featureList);  // recursive
-            featureList.addAll(ovl.getFeatures());
+         List<IOverlay> overlays = overlay.getOverlays();
+        if (overlays != null && !overlays.isEmpty()) {
+            for (IOverlay ovl : overlay.getOverlays()) {
+                getAllFeatures(ovl, featureList);  // recursive
+                featureList.addAll(ovl.getFeatures());
+            }
+        } else {
+            featureList.addAll(overlay.getFeatures());
```
Comment (jrummel) [inline sdk/sdk-api/src/main/java/mil/emp3/json/geoJson/GeoJsonExporter.java:479]: Does it enter this block if overlays is == null?

File: sdk/sdk-api/src/main/java/mil/emp3/json/geoJson/GeoJsonExporter.java (hunk 1 above)
Comment (rsupnekar) [reply sdk/sdk-api/src/main/java/mil/emp3/json/geoJson/GeoJsonExporter.java]: fixed

File: sdk/sdk-api/src/main/java/mil/emp3/json/geoJson/GeoJsonExporter.java (hunk 2 above)
Comment (rsupnekar) [reply sdk/sdk-api/src/main/java/mil/emp3/json/geoJson/GeoJsonExporter.java:479]: yes

Comment (jrummel) [reply sdk/sdk-api/src/main/java/mil/emp3/json/geoJson/GeoJsonExporter.java:479]: Ahh duh, I read overlay.getFeatures as overlays and figured a npe would occur.

**Review by jrummel — APPROVED** [review]

**Pull request merged**